src/core/segment_reader.rs

```diff
@@ -149,6 +150,21 @@ impl SegmentReader {
         }
     }
 
+    /// Accessor to the `BytesFastFieldReader` associated to a given `Field`.
+    /// May panic if the field is not a Bytes fast field.
```

Comment:
Does it?

Reply:
It doesn't. Will remove.

Reply:
Right now I think it does panic but in some specific corner case.. You can remove the panic if you address the other comment below.